armory/eval/evaluator.py

```diff
@@ -247,7 +247,7 @@ def run(
             except KeyboardInterrupt:
                 log.warning("Keyboard interrupt caught")
             finally:
-                log.warning("Shutting down container")
+                log.trace("Shutting down container")
```

Comment:
Can we put container ID or something in here to give reference...helpful when it doesn't shutdown cleanly

Reply:
done

Reply:
done